gem/Gemfile

```diff
@@ -16,8 +16,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
-gem 'coveralls', require: false
-gem 'rspec'
-gem 'rubocop'
-gem 'thor' # enable: bundle exec thor spec
```

Comment:
why did you delete the gems from the Gemfile? 😭 This broke the bundle spec command.

```
$ bundle exec thor spec
Could not find gem 'coveralls (~> 0.8.13)' in any of the gem sources listed in your Gemfile.
Run `bundle install` to install missing gems.
```